### DISCUSSION TITLE
使用 OpenIn 为你的链接设置专属浏览器 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#32] [Announcements]
# 使用 OpenIn 为你的链接设置专属浏览器 - forecho's Blog

引言
在日常使用中，你是否会需要这种情况，你使用的浏览器有两个帐户，一个是你的个人帐户，一个是你的工作帐户，工作帐户的 Chrome 登录的是工作的 GitHub 帐户，个人帐户的 Chrome 登录的是你私人的 GitHub 帐户，用于区分。
在工作沟通的 IM 软件中，你希望点击链接时，能够使用工作帐户的 Chrome 打开，而不是个人帐户的 Chrome 打开。不然同事发给你的私有仓库链接，你打开的时候，会提示你没有权限。
今天在 SetAPP 上我就发现了一款软件，叫做 OpenIn，可以帮助你解决这个问题。稍微配置一下，然后全自动完成你的需求。

https://blog.forecho.com/use-openin.html

**Comment by athurg** [comment]
工具是好工具，需求是真贴合需求。就是价格有点贵哦！折合人民币都快¥80了。

**Reply by forecho** [reply]
SetAPP 你值得拥有，200 左右人民币可以上车

**Reply by xyxc0673** [reply]
推荐一下免费的 Velja，可以实现这样的功能，而且还是可视化配置的，在 App Store 可以下载

**Reply by athurg** [reply]
Velja确实好用，赞
